IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_CableSegmentTypeCableSegment/DocPropertySet.xml

```diff
@@ -24,6 +24,16 @@
 		<DocProperty xsi:nil="true" href="SelfExtinguishing60332_3_2PP2O0qSSHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="HalogenProof_2Vyfs0qSSHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="FunctionReliable_2bJ_W0qSSHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="ACResistance_2dT1WHiH7Ge2PeqXa73Sp" />
+		<DocProperty xsi:nil="true" href="CurrentCarryingCapacity_2bRJDYSdX00x51ZO5f_6Kl" />
+		<DocProperty xsi:nil="true" href="DCResistance_1iHsZExLz0_hKDmfStAwc4" />
+		<DocProperty xsi:nil="true" href="MassPerLength_1bxC00qVOHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="MaximumCurrent_0w_s_0qUWHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="EffectiveBendingRadius_0lxvCy45jAwx0T2dIauLd" />
+		<DocProperty xsi:nil="true" href="FlameRetardantWireType_0smteSsUD7mPHMM2lHlTVl" />
```

Comment:
from the description this seems like a duplication of `SelfExtinguishing60332_1_2JRaK0qSSHuO00025QrEV` and `HalogenProof_2Vyfs0qSSHuO00025QrEV`